solr/solrj/src/java/org/apache/solr/client/solrj/impl/SolrClientCloudManager.java

```diff
@@ -47,7 +47,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Class that implements {@link SolrCloudManager} using a SolrClient */
+/**
+ * Class that implements {@link SolrCloudManager} using a SolrClient. TODO: Write a Http2 version of
+ * this class
+ *
+ * @deprecated Please look into using Solr's new Http2 clients
+ */
+@Deprecated(since = "9.0")
```

Comment:
Perhaps this class also can survive, if it is updated to use `CloudHttp2SolrClient`